dwave/optimization/model.pyx

```diff
@@ -2033,6 +2034,14 @@ cdef class ArraySymbol(Symbol):
 
         return self.array_ptr.size()
 
+    def sqrt(self):
```

Comment:
NumPy array's don't support `.sqrt()` so neither should we. Instead this should be a function in `mathematical.py` to match [np.sqrt](https://numpy.org/doc/stable/reference/generated/numpy.sqrt.html).